main.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html ng-app="App">
+    <meta name="viewport" content="width = device-width, initial-scale = 1">
+    <link rel="stylesheet" href="public/css/app.css">
+    <script src = "https://ajax.googleapis.com/ajax/libs/angularjs/1.4.9/angular.min.js"></script>
+    <script src="public/js/app.js"></script>
```

Comment:
Keep the file names consistent, lots of developers are picky about this, app.ctrl.js